spec/support/shared_examples/cnab400.rb

```diff
@@ -26,7 +26,7 @@
       {
         portfolio: '17777751042700080112',
         carteira: '1',
-        empresa_mae: 'asd',
+        empresa_mae: 'ASD',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.